core/src/main/java/edu/wpi/grip/core/sockets/SocketHints.java

```diff
@@ -75,6 +77,7 @@ private SocketHints() { /* no op */ }
 
   /* PRIVATE PARTIALLY CONSTRUCTED BUILDERS BELOW */
 
+  @SuppressFBWarnings(value = "UPM", justification = "False positive")
```

Comment:
"UMP"?

Reply:
**U**nused **P**rivate **M**ethod. This method is _clearly_ used elsewhere in this file, so it's a false positive

Reply:
Huh. Okay then. Thanks!